<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="CardWidget" name="ConnectionCard">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <spacer name="BottomVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="HorizontalSpacerSelectCOuntry_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="IconWidget" name="CountryIcon" native="true">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="icon" stdset="0">
             <iconset>
              <normaloff>:/images/alarms.png</normaloff>:/images/alarms.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BodyLabel" name="CurrentCountryText">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Russia</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="lightColor" stdset="0">
             <color>
              <red>96</red>
              <green>96</green>
              <blue>96</blue>
             </color>
            </property>
            <property name="darkColor" stdset="0">
             <color>
              <red>206</red>
              <green>206</green>
              <blue>206</blue>
             </color>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TransparentToolButton" name="ChooseServerButton">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>20</pointsize>
              <italic>false</italic>
              <kerning>true</kerning>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="HorizontalSpacerSelectCOuntry_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="HorizontalSpacerSelectCOuntry_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="BodyLabel" name="CurrentIPText">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="lightColor" stdset="0">
             <color>
              <red>96</red>
              <green>96</green>
              <blue>96</blue>
             </color>
            </property>
            <property name="darkColor" stdset="0">
             <color>
              <red>206</red>
              <green>206</green>
              <blue>206</blue>
             </color>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="HorizontalSpacerSelectCOuntry">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="HorizontalSpacerSelectCOuntry_8">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="ConnectBtn">
            <property name="font">
             <font>
              <family>HoloLens MDL2 Assets</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="ChangeIpBtn">
            <property name="font">
             <font>
              <family>HoloLens MDL2 Assets</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Change IP</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="HorizontalSpacerSelectCOuntry_9">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="TopVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TransparentToolButton</class>
   <extends>ToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
